### ISSUE TITLE
Figure out how to assign popups to points with the correct information

### ISSUE THREAD

**Issue by Bachibouzouk** [rl-institut/WAM_APP_FRED#31]
Would be cool to prepare this in the views.py and only use a function in the .js side

**Comment by Bachibouzouk** [comment]
@christian-rli - are you working on this today?

**Comment by Bachibouzouk** [comment]
This example defines a function to fetch data asynchronuously : 

http://bl.ocks.org/hpfast/5017e08b6c27b5abd287c2cf21c0c533

**Comment by christian-rli** [comment]
I meant to work on this today, but my setup doesn't seem to allow loading the geojson (from outside of the office). I changed some of my settings and at least got the app to start (which at first it also wouldn't do). Thanks for the link. I'm going to work on this in an independent map-sample and try to implement it on Monday. (I got one of those "does the server allow connections via port 5432" errors. I tried some port forwarding via the office, but to no avail)
